src/dp/distributions.rs

```diff
@@ -262,6 +262,8 @@ where
 }
 
 /// A DP strategy using the discrete gaussian distribution providing zero-concentrated DP.
+///
+/// This uses l2-sensitivity, with the substitution definition of neighboring datasets.
```

Comment:
I'm not sure what's more common, "l2" or "L2"
```suggestion
/// This uses L2-sensitivity, with the substitution definition of neighboring datasets.
```